core/src/avm1/globals/external_interface.rs

```diff
@@ -64,23 +64,16 @@ pub fn call<'gc>(
     };
     let name = name.coerce_to_string(activation)?;
 
-    if let Some(method) = activation
-        .context
-        .external_interface
-        .get_method_for(&name.to_utf8_lossy())
-    {
-        let mut external_args = Vec::with_capacity(external_args_len);
-        if external_args_len > 0 {
-            for arg in &args[1..] {
-                external_args.push(ExternalValue::from_avm1(activation, arg.to_owned())?);
-            }
+    let mut external_args = Vec::with_capacity(external_args_len);
```

Comment:
I would probably write this using an iterator an map.

Reply:
This was old code moved up, so I didn't really touch it - but good idea, changed :)